### PR TITLE
tails_printing_guide.rst: fix style issues

### LINKED ISSUE #3013: docs: proofread tails_printing_guide.rst for typos and style problems
## Description

Read https://github.com/freedomofpress/securedrop/blob/develop/docs/tails_printing_guide.rst and look for:

* typos
* [violations of the documentation guidelines](https://docs.securedrop.org/en/latest/development/documentation_guidelines.html)

Rewording for clarity or adding content is not in the scope of this issue. When a typo or style problem is found, open a pull request fixing it and make sure to reference this issue so it is closed.

If no typo or style problem is found, add a comment to this issue. It will be closed after a few person report they did not find an issue.

### PR DESCRIPTION
**Pull request by prakashdanish**
## Status

Ready for review

## Description of Changes

Related to #3013.

Styling changes.

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/tails_printing_guide.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ a printer, which will show the following:
 
 |add printer|
 
-Click "Add a Printer". By doing so, you'll now get a list of printers that Tails
+Click **Add a Printer**. By doing so, you'll now get a list of printers that Tails
```
Comment (kushaldas) [inline docs/tails_printing_guide.rst]: This looks good.

File: docs/tails_printing_guide.rst [hunk 2]
```diff
@@ -75,7 +75,7 @@ following:
 
 This indicates the Tails is attempting to install the USB printer; assuming you
 receive no errors in this process, you will then see the following screen,
-which indicates that the printer is "ready" for printing.
+which indicates that the printer is **ready** for printing.
```
Comment (kushaldas) [inline docs/tails_printing_guide.rst]: Not sure if this should be in bold.

Comment (heartsucker) [reply docs/tails_printing_guide.rst]: Agree. I think this emphasis should be removed.

Comment (prakashdanish) [reply docs/tails_printing_guide.rst]: ok, will revert it back later today. thanks for the review.

**Comment by dachary** [comment]
@kushaldas is your concern addressed ?

**Comment by dachary** [comment]
@prakashdanish force-repushed to your branch after a rebase to address a CI issue that is unrelated to your work.

**Pull request merged**